tcl/main.tcl

```diff
@@ -1233,7 +1232,7 @@ proc CreateGameInfo {} {
           -variable gameInfo(showFEN) -offvalue 0 -onvalue 1 -command updateBoard
 
   .main.gameInfo.menu add checkbutton -label GInfoMarks \
-          -variable gameInfo(showMarks) -offvalue 0 -onvalue 1 -command updateBoard
+          -variable ::board::_showMarks(.main.board) -offvalue 0 -onvalue 1 -command updateBoard
```

Comment:
Neither version is good :)

::board::_ showMarks is a private variable of the board class, so V1 is not acceptabled because other boards would be affected as well when :: gameInfo (showMarks) changes.

The problem with V2 is the reverse: a private variable is accessed outside the class.

Instinctively I would say the simplest solution is to just change the -command of this line so that it invokes ::board::showMarks before updateBoard